README.md

```diff
@@ -89,6 +89,25 @@ your project, as described above.
 
 ### Configuring your Engine
 
+Engines depend on `ember-engines` as a peer dependency, which means
+that it must be installed in the consuming app. So make sure that it's
+specified as such in your engine's `package.json`:
+
+```json
+  "peerDependencies": {                                                                                                   
+    "ember-engines": "*" 
+  },
+```
+
+Also, make sure to add `ember-engines` as a dev dependency. This allows local
```

Comment:
This should specify into which `package.json` and it should. We should also clarify how to do this for in-repo vs. standalone engines.